portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SitesActivities.testcase

```diff
@@ -113,6 +113,82 @@ definition {
 		}
 	}
 
+	@description = "Feature ID: Legacy | Automation ID: LRAC-10257 | Test Summary: Assert total amount of views in all cards"
+	@ignore = "true"
+	@priority = "4"
+	test AssertTotalAmountOfViewsInAllCards {
+		ACDXPSettings.connectDXPtoAnalyticsCloud(siteName = "Site Name");
+
+		task ("Add a new user in DXP") {
+			JSONUser.addUser(
+				userEmailAddress = "ac@liferay.com",
+				userFirstName = "ac",
+				userLastName = "ac",
+				userScreenName = "ac");
+		}
+
+		task ("Sync Contact Data") {
+			ACDXPSettings.syncNewContactData();
+		}
+
+		task ("Login new user and view page") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "ac@liferay.com",
+				userLoginFullName = "ac ac");
+		}
+
+		task ("Interation with page") {
+			ACUtils.navigateToSitePageWithUser(
+				pageName = "AC Page",
+				siteName = "Site Name");
+
+			ACUtils.closeAllSessionsAndWait();
+		}
+
+		task ("AC launch") {
+			ACUtils.launchAC();
+
+			ACProperties.switchProperty(propertyName = "${propertyName}");
+		}
+
+		task ("Asset views for 24 hours") {
+			ACNavigation.goToSites();
+
+			ACNavigation.switchTab(tabName = "Pages");
+
+			ACTimeFilter.clickTimeFilterButton();
+
+			ACTimeFilter.setLast24Hours();
+
+			ACNavigation.openItem(itemName = "AC Page - Site Name - ${dataSourceName}");
+		}
+
+		task ("Get views value") {
+			var current = ACSettings.getCurrentNumber(metricName = "Views");
+
+			ACTimeFilter.clickTimeFilterButton();
+
+			ACTimeFilter.setLast24Hours();
+		}
+
+		task ("Assert views by location") {
+			var country = ACUtils.getIPCountry();
```

Comment:
Hey @calvinctvo, I think this macro still doesn't work to get the correct continent (I see that all tests that use it are disabled). I think if it doesn't work you could pass "Local Network" which is the continent that appears when we run locally as it happens in CI. Does the macro work for you? If not what do you think of my suggestion?

Reply:
Hey @ViniLopes20 , if the correct IP continent doesn't work, would this test still fails even if I change to "Local Network". I'm thinking of putting this test on hold for this reason. What do you think?

Reply:
Hey @calvinctvo, I don't understand why the test would fail if you used "Local Network"? When we run the test locally or in the CI, the country that is shown is Local Network. Do you think it wouldn't work?


I think it would work like this, you would say that the Country would be Local Network and you wouldn't need to use the macro


`
ACCards.viewGeomapTableText(
	  cardTitle = "Sessions by Location",
	  leftText = "Local Network",
	  rightText = "${current} 100%");
`

Reply:
hey @ViniLopes20 , I've send in the fix that you suggested. Could you take a look when you have time? Thank you!